cogs/quotes.py

```diff
@@ -466,6 +538,77 @@ async def generate(self, ctx, seed: str = None, min_length: int = 1):
 
             await ctx.send(' '.join(longest_sentence))
 
+    @commands.command()
+    @is_moderator()
+    async def purgequotes(self, ctx, user: str):
```

Comment:
can you make this `purge_quotes` and add an alias `purgequotes` to keep the function names pep8?